chart/templates/mayastor/obs/stats-configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.eventing.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-event-store
+  labels:
+    app: event-store
+    {{ include "label_prefix" . }}/release: {{ .Release.Name }}
+    {{ include "label_prefix" . }}/version: {{ .Chart.Version }}
+data:
+   stats: '{"pool":{"pool_created":0,"pool_deleted":0},"volume":{"volume_created":0,"volume_deleted":0}}'
```

Comment:
must we include the stats here? Otherwise we need to update when the schema changes and IIRC we didn't want to have to manage changes etc